static/js/controllers/associations.controller.js

```diff
@@ -6,14 +6,10 @@ var Request = require('request');
 exports.getFromTables = (req, res) => {
     var studyIDObjs = req.body.studyIDObjs
     var refGen = req.body.refGen;
-    var defaultSex = req.body.sex;
+    var sexes = req.body.sexes;
```

Comment:
"sexes" should be "sex" for both the associations table and the study table

Reply:
I'll make sure that is updated in the model. Here I think it is okay because the user could request to filter by Male and Female